crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/account_deposit.rs

```diff
@@ -0,0 +1,107 @@
+use crate::prelude::*;
+use std::collections::HashMap;
+
+/// Represents an account deposit, which includes specified and unspecified resources.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Record)]
+pub struct AccountDeposit {
+    specified_resources: HashMap<ResourceAddress, SimpleResourceBounds>,
```

Comment:
I prefer using `IndexMap` since order is always the same and it makes unit testing a bit easier. But nothing I feel too strongly about.